### PR TITLE
Expose Subscription component

### PR DESCRIPTION
**Pull request by datenreisender**
Needed for NordicPlayground/nrfcloud-web-frontend#360, this exposes the `Subscription` component.

@coleHafner Do you have a suggestion how I should apply the new versioning from #12 on this branch? My idea: After merging this branch into master I then rebuild the `dist/aws-wrapper{.min,}.js` and tag a version 1.1.0 on master afterwards. Do you agree?

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1,17 +1,5 @@
...
-	ApolloProvider,
-	Query,
-	Mutation,
-	graphql,
-	gql
-};
+export { default as AWSAppSyncClient } from 'aws-appsync';
+export { default as Cognito } from './Cognito';
+export { isDevZoneLogin } from './DevzoneHelper';
+export { ApolloConsumer, ApolloProvider, Query, Mutation, Subscription, graphql } from 'react-apollo';
```
Comment (coleHafner) [inline src/index.ts]: Please break these out to one line each when the imports are > 2. This is not an existing rule, I don't think, but it makes it more readable, IMO. Thoughts, @datenreisender ?

**Comment by coleHafner** [comment]
> @coleHafner Do you have a suggestion how I should apply the new versioning from #12 on this branch? My idea: After merging this branch into master I then rebuild the `dist/aws-wrapper{.min,}.js` and tag a version 1.1.0 on master afterwards. Do you agree?

Agreed, @datenreisender. Test the build locally and when you're satisfied with it open a PR, like you did already, merge it to master, tag it, rebuild it (and copy to iris FE), and push the whole thing to master (and push tags too). These steps [are outlined in the README](https://github.com/nRFCloud/aws-client#deploying).

File: src/index.ts (hunk 1 above)
Comment (coleHafner) [inline src/index.ts]: Please break these out to one line each when the imports are > 2. This is not an existing rule, I don't think, but it makes it more readable, IMO. Thoughts, @datenreisender?

Comment (datenreisender) [reply src/index.ts]: I am fine with breaking up the line and did so in 4b3d898. Even nicer would be if we had an automatic formatter for these things.

**Comment by coleHafner** [comment]
Given the current discussions going on about `react-apollo`, I suggest freezing this PR until we can all come to a resolution. 

My two cents: 
- If we are using GraphQL, we should use its corresponding components (Query, Mutation, Subscription). That way we can depend on the community to develop and maintain these for us, instead of doing it ourselves. It will allow us to focus on features that our important to our app instead of maintaining solutions for the underlying technology. 

- Personally I feel like using GraphQL provides little benefit. It's like having an API for our API. It's corresponding components come with a lot of bells and whistles that we didn't ask for and seem to require extra configuration. It's a fixed cost, but it's one that we didn't have before with our simple REST API. Along the same lines, if something with the GQL components break, I would have no idea where to start. I'd figure it out, but, once again, this is not an issue we have with our existing REST API. Perhaps I haven't used it enough to realize its full potential. 

I don't recall any discussion regarding using GraphQL, it just kind of showed up one day. This happens with a lot of things in the project. There's little to no discussion regarding decisions around the project. It's an indication of our lack of communication as a team. I know it takes longer, especially with the timezone difference, but it's sure a lot easier to move forward when we're not all arguing and bickering.

**Comment by coleHafner** [comment]
Problem
We need to discuss solutions and new technologies before using them. 

Proposed Solution 
Since meetings are expensive and rarely productive, why don't we have a Slack channel where the person proposing the new change or technology can pitch their idea to the rest of the team? That way we can all discuss it our leisure and gauge whether or not the solution is feasible. Granted, not all proposal can be resolved via Slack, but I bet the majority could be.

**Comment by datenreisender** [comment]
@coleHafner Let us see, whether we can find a decision in the next two days. If we do not reach a decision by then, I would ask you to unfreeze this PR so I can continue with my work without being to blocked. If we later decice against the use of Apollo, we can easily remove all references to Apollo from the `aws-client` again.

**Comment by coleHafner** [comment]
Okay, @datenreisender. I added a proposal doc to the Slack conversation that hopefully will help us move forward with a resolution.

**Comment by datenreisender** [comment]
(╯°□°）╯︵ ┻━┻ THIS IS RIDICULOUS

**Pull request closed without merging**